### DISCUSSION TITLE
Zero conf channel opening failing.

### DISCUSSION THREAD

**Discussion by 0xtav** [lightningnetwork/lnd#6948] [Troubleshooting] [answered]
### Background

I'm trying to open a zero-conf Lightning channel and the counterpart is rejecting the channel opening, returning this error:
```
lncli openchannel 0223c901127aad49ee60f34062d4a506d3871081b2074b425a5a8c886e79dc4a06 --local_amt 60000 --push_amt 50000 --zero_conf --channel_type anchors --private
[lncli] rpc error: code = Unknown desc = received funding error from 0223c901127aad49ee60f34062d4a506d3871081b2074b425a5a8c886e79dc4a06: chan_id=062e2e0dd64269d73a00f6d586596789d53653845836fb3810f16209e6d55778, err=channel rejected
```

### Environment

* version of `v0.15.1-beta`
* version of `bitcoind-v23.0.0`

**Comment by alexbosworth** [comment]
Do they have a channel acceptor

**Comment by 0xtav** [comment]
> Do they have a channel acceptor

How can I configure this?

**Comment by alexbosworth** [comment, marked as answer]
there is a doc here https://github.com/lightningnetwork/lnd/blob/master/docs/zero_conf_channels.md

**Comment by 0xtav** [comment]
> there is a doc here https://github.com/lightningnetwork/lnd/blob/master/docs/zero_conf_channels.md

This documentation was not very clear to me, how will the other party accept the opening of the channel?

**Comment by alexbosworth** [comment]
They would use the channel acceptor API to accept it

**Comment by 0xtav** [comment]
> 

So would the other party stream this endpoint and confirm the opening through that?https://api.lightning.community/?python#v1-channels-acceptor

**Comment by alexbosworth** [comment]
Yeah they must explicitly accept it if it's zero conf

**Comment by saubyk** [comment]
Converting this issue to a discussion
